nselib/data/http-sql-errors.lst

```diff
@@ -56,33 +64,48 @@ PHP Warning
 Parent Directory
 Permission denied: \'GetObject\'
 PostgreSQL query failed: ERROR: parser: parse error
+SQL Error
 SQL Server Driver%]%[SQL Server
+SQL Server JDBC Driver
 SQL command not properly ended
 SQLException
+Supplied argument is not a valid MySQL result resource in
 Supplied argument is not a valid PostgreSQL result
 Syntax error in query expression
+System.IndexOutOfRangeException
 The error occurred in
 The script whose uid is
 Type mismatch
 Unable to jump to row
-Unclosed quotation mark before the character string
+Unable to select * from
+Unclosed quotation mark
+Unknown column
 Unterminated string constant
 Warning: Cannot modify header information %- headers already sent
 Warning: Supplied argument is not a valid File%-Handle resource in
 Warning: mysql_query%(%)
 Warning: pg_connect%(%): Unable to connect to PostgreSQL server: FATAL
-You have an error in your SQL syntax near
+Warning: require(%)
+array_merge()
```

Comment:
Parentheses here and throughout must be %-escaped. https://www.lua.org/manual/5.3/manual.html#6.4.1